cmake/ConfigureCompiler.cmake

```diff
@@ -23,6 +23,8 @@ env_set(USE_LIBCXX "${_use_libcxx}" BOOL "Use libc++")
 static_link_libcxx(_static_link_libcxx)
 env_set(STATIC_LINK_LIBCXX "${_static_link_libcxx}" BOOL "Statically link libstdcpp/libc++")
 env_set(TRACE_PC_GUARD_INSTRUMENTATION_LIB "" STRING "Path to a library containing an implementation for __sanitizer_cov_trace_pc_guard. See https://clang.llvm.org/docs/SanitizerCoverage.html for more info.")
+env_set(GENERATE_INSTR_PROFILE OFF BOOL "If set, build FDB as an instrumentation build to generate profiles")
```

Comment:
Hmm, no, I still think we should use something like
`PROFILE_INSTR={OFF, GENERATE, USE}` and `PROFILE_INSTR_FILE=<file>`.
So you cannot set both `use` and `generate` flags accidentally.
Also, this way, you can provide a way to specify the file name for both generate and use.
```
-fprofile-instr-generate=<file>
-fprofile-instr-use=<value>
```

Reply:
I can add a check in cmake file, then it will fail if user set both use and generate flags accidentally.

Your proposal is more reasonable than the current interface, however I still have one concern, the current interface is just a thin wrapper of clang interface, and your proposal is totally different from clang interface, will it confuse users?

Reply:
That's a valid argument. Then, let's use `PROFILE_INSTR_GENERATE` and `PROFILE_INSTR_USE` to match with the clang option names.

Reply:
It's done, please take another look.